src/blockchain.js

```diff
@@ -66,16 +79,27 @@ export default class Blockchain {
         return textToSafeJson(response, keys);
     }
 
+    async getActiveLeaseList(address, key) {
```

Comment:
remove `key` here. We should save the key in Blockchain object construction

```
let chain = new vsys.Blockchain(node_address, network_byte, api_key);
```

And `api_key` can be omitted. The default value is empty string if not pass `api_key`.